### PR TITLE
Vupirov added TypeScript support

### PR DESCRIPTION
**Pull request by Valodya**

### CODE REVIEW

File: libs/backendless.js [hunk 1]
```diff
@@ -3844,132 +3850,139 @@
...
+        this.selector = args.selector || undefined;
+    };
+
+    root.GeoPoint = Backendless.GeoPoint = GeoPoint;
+    root.Bodyparts = Backendless.Bodyparts = Bodyparts;
+    root.GeoCluster = Backendless.GeoCluster = GeoCluster;
+    root.PublishOptions = Backendless.PublishOptions = PublishOptions;
+    root.DeliveryOptions = Backendless.DeliveryOptions = DeliveryOptions;
+    root.SubscriptionOptions = Backendless.SubscriptionOptions = SubscriptionOptions;
+    root.BackendlessGeoQuery = Backendless.BackendlessGeoQuery = BackendlessGeoQuery;
```
Comment (vengrov) [inline libs/backendless.js]:
```
root.BackendlessGeoQuery = Backendless.GeoQuery = GeoQuery;
```

Comment (Valodya) [inline libs/backendless.js]: fixed

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backendless",
-  "version": "3.0.11",
+  "version": "3.1.0",
```
Comment (vengrov) [inline package.json]: Please revert this change. We don't use semantic versioning here
